website_form_recaptcha/models/website_form_recaptcha.py

```diff
@@ -16,15 +16,18 @@ class WebsiteFormRecaptcha(models.AbstractModel):
     _description = 'Website Form Recaptcha Validations'
     URL = 'https://www.google.com/recaptcha/api/siteverify'
     RESPONSE_ATTR = 'g-recaptcha-response'
-    ERROR_MAP = {
-        'missing-input-secret': 'The secret parameter is missing.',
-        'invalid-input-secret':
-            'The secret parameter is invalid or malformed.',
-        'missing-input-response': 'The response parameter is missing.',
-        'invalid-input-response':
-            'The response parameter is invalid or malformed.',
-        None: 'There was a problem with the captcha entry.',
-    }
+
+    @api.model
+    def _get_error_map(self):
```

Comment:
It seems more sensible IMHO to have a `_get_error_message(self, errorcode)` method instead. Not a requirement.

Reply:
You're absolutely right :)